packages/app-forms/src/admin/plugins/routes.js

```diff
@@ -24,7 +24,7 @@ export default [
                 render={() => (
                     <SecureRoute roles={["form-editors-editor"]}>
                         <Helmet>
-                            <title>Edit form</title>
+                            <title>Edit form builder</title>
```

Comment:
Please remove the "builder" word here, we're not really editing form builder, but form. :)

Reply:
Got it, i have removed